.github/workflows/teste2e.yml

```diff
@@ -1,9 +1,19 @@
 name: Run E2E Test for skype-sender lambda
 on:
   push:
-    branches: master
-  pull_request:
-    branches: master
+    branches:
+      - master
+    paths:
+      - '.github/workflows/**'
+      -  'conf/**'
+      -  'daily_messages/**'
+      -  'dummy_lambda/**'
+      -  'holiday_reminder/**'
+      -  'pto_detector/**'
```

Comment:
Again, why should daily messages end to end test run if we change pto-detector lambda?